pkg/services/store/auth.go

```diff
@@ -47,5 +47,8 @@ func GetUserIDString(user *user.SignedInUser) string {
 	if user.IsRealUser() {
 		return fmt.Sprintf("user:%d:%s", user.UserID, user.Login)
 	}
+	if user.UserID == 0 {
+		return "sys:0:admin" // default is ?
```

Comment:
should this check for anonymous users?

semi-related: i opened a PR to migrate to appcontext: https://github.com/grafana/grafana/pull/58157